biletmajster-backend/Controllers/EventApi.cs

```diff
@@ -143,6 +149,121 @@ public virtual async Task<IActionResult> CancelEvent([FromRoute][Required] strin
+            {
+                return StatusCode(403, new ErrorResponse { Message = "invalid session" });
+            }
+            if (!await _modelEventRepository.UpdateEvent(@event))
+            {
+                return StatusCode(403, new ErrorResponse { Message = "invalid session" });
+            }
+            return StatusCode(204);
+        }
+
+        /// <summary>
+        /// Get list of photo of event
+        /// </summary>
+        /// <remarks>Returns a list of photo paths</remarks>
+        /// <param name="id">ID of event to return</param>
+        /// <response code="200">successful operation</response>
+        /// <response code="400">Invalid ID supplied</response>
+        /// <response code="404">Event not found</response>
+        [HttpGet]
+        [Route("/events/{id}/photos")]
+        [ValidateModelState]
+        [SwaggerOperation("GetPhoto")]
+        [SwaggerResponse(statusCode: 200, type: typeof(List<string>), description: "successful operation")]
+        public virtual async Task<IActionResult> GetPhoto([FromRoute][Required] long? id)
+        {
+            if (id == null)
+            {
+                return StatusCode(400, new ErrorResponse { Message = "Invalid ID supplied" });
+            }
+            var @event = await _modelEventRepository.GetEventByIdAsync((long)id);
+            if (@event == null)
+            {
+                _logger.LogDebug($"Event with id {id} not found");
+                return StatusCode(404, new ErrorResponse { Message = "Event not found" });
+            }
+            var list = new List<string>();
+            @event.EventPhotos.ForEach(x => list.Add(x.DownloadLink));
+            return StatusCode(200, list);
+        }
+        /// <summary>
+        /// patch existing event
+        /// </summary>
+        /// <param name="id">id of Event</param>
+        /// <param name="path">path of photo</param>
+        /// <response code="200">path added</response>
+        /// <response code="400">path already exist</response>
+        /// <response code="403">invalid session</response>
+        /// <response code="404">id not found</response>
+        [HttpPost]
+        [Route("/events/{id}/photos")]
+        [Authorize]
+        [ValidateModelState]
+        [SwaggerOperation("PutPhoto")]
+        [SwaggerResponse(statusCode: 200, type: typeof(EventPhotosDTO), description: "successful operation")]
+        public virtual async Task<IActionResult> PutPhoto([FromRoute][Required] string id, [FromForm][Required] IFormFile file)
+        {
+            var @event = await _modelEventRepository.GetEventByIdAsync(long.Parse(id));
+            if (@event == null)
+            {
+                _logger.LogDebug($"Event with id: {id} not found");
+                return StatusCode(404, new ErrorResponse { Message = "Event not found" });
+            }
+            var Blobresult = await _blobStorage.UploadFileAsync(file, file.FileName);
```

Comment:
I strongly suggest giving the file a random name + preserve extension, like:
`photo.jpg` => `06be32db-e5b9-4092-9f4d-f0ac11de67ed.jpg`
`photo1.bmp` => `66c41062-96bc-4586-bd77-54d21d5bdfc0.bmp`
Right now, if both users want to add a file `photo.jpg`, they can't do that.